rest/data/interface.go

```diff
@@ -190,6 +190,8 @@ type Connector interface {
 
 	// FindRecentTasks finds tasks that have recently finished.
 	FindRecentTasks(int) ([]task.Task, *task.ResultCounts, error)
+	FindRecentTaskListDistro(int) (*task.ResultCountList, error)
+	FindRecentTaskListProject(int) (*task.ResultCountList, error)
```

Comment:
it's arguably a bug that we ever return database models from the Connector interface... it's probably cleaner to return APIModels...

Reply:
done.